docs/concepts/trading-on-vega/settlement.md

```diff
@@ -0,0 +1,85 @@
+---
+sidebar_position: 6
+title: Settlement
+hide_title: false
+---
+Settlement is the process of moving collateral between accounts when a position is closed, when the market's mark price changes, and when the market expires. Vega operates as a decentralised 'central counterparty' that facilitates settlement.
+
+More specifically, settlement on markets occurs when:
+* A position is fully or partially closed, with settlement occuring for the closed volume. An open position is closed when the open position's holder enters into a countertrade
+* The protocol runs mark to market settlement
+* A market expires, at which point all open positions in the market are settled
+
+## Parameters defined through governance
+Market proposals define the fundamental details about a market, including its settlement asset and how it can terminate and be settled.
+
+### Settlement asset
+Vega can be used to create cash-settled futures markets, meaning they are margined and settled in a single asset. 
+
+On those markets, settlement occurs in the settlement asset of the market, which is defined in the governance proposal that led to the market's creation. The settlement asset **does not** need to be the same asset as the ‘quote unit’ (i.e. ETH on a BTC/ETH December 2028 market).
+
+### Data for settlement trigger
+In addition to the settlement asset, a market's proposal also specifies the data sources that determine settlement triggers for the end of a market's lifecycle. The data sources defined, and the filters applied to them, must be able to provide the market's expiry date/time and its final price/value at expiry. 
+
+At the point when a market arrives at its specified expiry, trading terminates, and then the data sources provide the final settlement value. The protocol settles all open positions on the market using the price provided by the data source.
+
+<!--
+:::note Read more
+[Data sources]
+:::
+-->
+ 
+## Settlement at market expiry
+After a market reaches its maturity date and time, a final settlement is carried out. That settlement is dependent on the pre-defined data source publishing data that triggers the market’s expiry.
+
+The trigger will cause the market to enter into a 'trading terminated' state. Markets in this state no longer accept trading, but retain the positions and margin balances that were in place after processing the market’s maturity trigger. Termination occurs just prior to settlement, so that no trading can occur after the settlement result can be known by participants.
+
+As positions are regularly mark to market settled, the final settlement is, in effect, a final mark to market settlement using the data source price.
```

Comment:
I would prefer "data source quote" to "data source price". But for futures it basically doesn't matter.